gulp/tasks/scripts.js

```diff
@@ -228,13 +246,14 @@ gulp.task( 'scripts',
     'scripts:modern',
     'scripts:apps',
     'scripts:external',
-    'scripts:ondemand'
+    'scripts:ondemand',
+    'scripts:admin'
   )
 );
 
 gulp.task( 'scripts:watch', function() {
   gulp.watch(
     configScripts.src,
-    gulp.parallel( 'scripts:modern' )
+    gulp.parallel( 'scripts:modern', 'scripts:admin' )
```

Comment:
```suggestion
    gulp.parallel( 'scripts:modern' )
```

I'm sure this was helpful while you were iterating, but let's remove it before merging this PR, since working on admin scripts is a pretty rare use case.

Or, alternatively, could make a `scripts:watch:admin` variant.